ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/UpdatedMetaDataTracker.java

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
There are other test cases to see if replication works for dynamic inserts. So, this test case is intended to check if INSERT event is not generated for dynamic inserts.